chat/public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+    <input type="submit" id="button" />
+    <div>
+      <pre id="chatid"/>
+ </div>
+ <script>
+const button =document.getElementById("button");
+button.onclick=()=>{
+  const name=document.getElementById("Name").value;
+  const comment=document.getElementById("Comment").value;
+  const xhttp=new XMLHttpRequest();
+  xhttp.open('GET', `/anyurl?Name=${name}&Comment=${comment}`);
```

Comment:
change name of request

Reply:
hey i changed it ,cna you please review it